build.rs

```diff
@@ -76,6 +76,10 @@ fn main() {
     }
   }
 
+  if cfg!(target_arch = "x86_64") {
```

Comment:
For some reason this doesn't work like I expect. Instead of only enabling `use-snapshots` only for x86_64 builds it seems to be enabling them for all builds.